meta/core.json

```diff
@@ -48,5 +48,6 @@
             "type": "string",
             "format": "uri-reference"
         }
-    }
+    },
+    "default": true
```

Comment:
I don't agree with having defaults in the roots of the meta-schemas.  What's the motivation for this?

Reply:
I get that the motivation is to avoid putting `"default": true` everywhere we reference a schema, but the default isn't always "true" for all references.

Traditionally, in our meta-schemas, `default` has meant something like: if this doesn't exist, the validation result would be the same as if it did exist and had this default value. For example, the `default` of `properties` is `{}`. Therefore, `{}` has the same result as `{ "properties": {} }`.

So, now consider the `not` keyword. It references the top-level meta-schema. If the top-level meta-schema has `"default": true` that says the `default` of `not` is `true`. Therefore, `{}` has the same result as `{ "not": true }`, which is not correct. That means each time something references the top-level meta-schema it needs to define a `default` that makes sense in its context.

... or just get rid of `default` in meta-schemas altogether because it doesn't provide any real value.

Reply:
@jdesrosiers I _think_ we're arguing the same side of the coin.

Reply:
Yes, we are. Bottom line is that `default` doesn't belong at the root of meta-schemas.

Reply:
> So, now consider the not keyword. It references the top-level meta-schema. If the top-level meta-schema has "default": true that says the default of not is true. Therefore, {} has the same result as { "not": true }, which is not correct. That means each time something references the top-level meta-schema it needs to define a default that makes sense in its context.

Or, we can add a default for not -- which should be false.  that is, `..., "not": { "$dynamicRef": "#meta", "default": false }, ...`

So, how about we remove all uses of default where it's just the obvious thing (`{}` for objects, or `true`or `{}` for schemas), and keep them only when they're non-standard -- which is, I believe, just "not", "deprecated", "readOnly", "writeOnly", "uniqueItems", and "minContains".)

If that's amenable to all I can amend this PR.